### DISCUSSION TITLE
Pretty printing XML Namespace response

### DISCUSSION THREAD

**Discussion by orcishbolo** [filiphric/cypress-plugin-api#105] [General]
Hi Filip, the response to this request was printed straight line. It was an XML Namespace. Can we display it as prettier?

![image](https://user-images.githubusercontent.com/43512975/213083937-b9bd5eba-641b-448f-a205-b6c8351a5bb8.png)

**Comment by filiphric** [comment]
I’ll take a look into some solutions, it can probably be done somehow
